### PR TITLE
Added check to find if .qsub.out files are existing

### PR DESCRIPTION
**Pull request by lalet**
The checksums-before threw error since we missed to put a check before grep on .qsub.out files

### CODE REVIEW

File: checksums.sh [hunk 1]
```diff
@@ -15,18 +15,19 @@ then
...
+QSUB_FILE=".qsub.out.*"
 
-echo "###########################################"
-echo "############ Checksum of docker image #####"
-echo "###########################################"
-
-grep "Digest: sha256:" .qsub.out.*
+count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
+if [ $count != 0 ]
+then
```
Comment (glatard) [inline checksums.sh]: You could just do `ls ${QSUB_FILE} && grep "Digest ...`
ls $QSUB_FILE will fail if there is no file matching the pattern and `&&` won't execute the grep. Question: should we stop the script if the file is not there, i.e., the container checksum can't be obtained?

Comment (lalet) [reply checksums.sh]: If the file is already existing in the docker images repo, would we get a checksum in the .qsub.out files ? If not then we need not stop I think. Thoughts ?

Comment (lalet) [reply checksums.sh]: We should stop as you said. Even if the image is not getting pulled , the sha digest in shown in the .qsub.out files.

> CBRAIN Task Starting At 1483719502 : 2017-01-06 11:18:22
docker pull bigdatalabteam/hcp-prefreesurfer:exec-centos5-fslbuild-centos5-latest
exec-centos5-fslbuild-centos5-latest: Pulling from bigdatalabteam/hcp-prefreesurfer
Digest: sha256:f0320a72501b0b8cf25ebaec11801a7d1d7e55857fcb699b57aa433d0d6d65e9
Status: Image is up to date for bigdatalabteam/hcp-prefreesurfer:exec-centos5-fslbuild-centos5-latest

So yes we should stop execution.

File: checksums.sh [hunk 2]
```diff
@@ -17,17 +17,11 @@ fi
...
 
-count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
-if [ $count != 0 ]
-then
-    echo "###########################################"
-    echo "############ Checksum of docker image #####"
-    echo "###########################################"
-    grep "Digest: sha256:" $QSUB_FILE
-fi
+(ls ${QSUB_FILE} && grep "Digest: sha256:" ${QSUB_FILE}) || (die ${SUBJECT_FOLDER} "Cannot find sha256 digest of docker image")
```
Comment (glatard) [inline checksums.sh:20]: OK, but don't you want to keep the ****** banner?

**Review by glatard — APPROVED** [review]

File: checksums.sh (hunk 2 above)
Comment (lalet) [reply checksums.sh:20]: Here even if it is empty the banner would get printed. Is that the normal convention ?

Comment (glatard) [reply checksums.sh:20]: well that's what's done with the `find` command below anyways.

**Pull request merged**